apps/src/aichat/views/ChatWarningModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import aichatI18n from '@cdo/apps/aichat/locale';
```

Comment:
Out of curiosity - if you change this to `const aichatI18n = require('@cdo/aichat/locale')`, does the storybook work?

Reply:
Tried that locally - The error will look prettier, but there'll still be an error :(
<img width="1678" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/22244040/ce592830-6642-477d-a83f-c47638ce0daf">